src/propertyvalueestimate.py

```diff
@@ -5,13 +5,11 @@
 from selenium.webdriver.support import expected_conditions as EC
 from bs4 import BeautifulSoup
 import domainapi as api
+from domainapi import DOMAIN_CLIENT_URL
 
-advertId = 2017019671
-DOMAIN_CLIENT_URL = "https://www.domain.com.au/"
-AD_ENDPOINT = DOMAIN_CLIENT_URL + str(advertId)
```

Comment:
This isnt a constant so shouldn't be capital, it should be generated within the funciton, see how i did it